pytorch_to_returnn/torch/nn/modules/module.py

```diff
@@ -623,15 +626,20 @@ def check_call_returnn_outputs_to_prev_torch(call: CallEntry, *, update_ops: Lis
       idx_repr = f" {i + 1}/{len(call.outputs_flat)}" if len(call.outputs_flat) > 1 else ""
       print(f"**** validate: add call {call} output{idx_repr} tensor {x}")
     out, _ = session.run((
-      [(x.returnn_data.placeholder, x.returnn_data.size_placeholder.as_dict()) for x in call.outputs_flat], update_ops),
+      [(x.returnn_data.placeholder, x.returnn_data.size_placeholder.as_dict()) if x is not None else tf.constant(0) for
```

Comment:
```suggestion
      [(x.returnn_data.placeholder, x.returnn_data.size_placeholder.as_dict()) if x is not None else {} for
```

We could do this change to avoid unnecessary ops in the graph, right?

Reply:
Ah yes, does that work? If it works, then sure, much better. Or maybe empty tuple instead.

Reply:
Yes, both work.